cmd/sonarExecuteScan.go

```diff
@@ -137,6 +137,9 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 		}
 		sonar.addOption("sonar.projectVersion=" + version)
 	}
+	if GeneralConfig.Verbose {
```

Comment:
Maybe this needs to be part of : 
https://github.com/SAP/jenkins-library/blob/c2bfacd58c5d6f7e4ee4ea1af60a78b4bdc66828/cmd/sonarExecuteScan_generated.go#L24
I guess that's why integration tests are failing ?

Reply:
If I run a go generate on this branch, it doesn't change any of the generated files so I think maybe it's not that. I thought maybe it was the changed unit test, but I ran the test on my local machine and it worked too...

Reply:
The verbose flag already exists for Piper, should not have to be added to any individual step. I doubt the ITs are failing due to that